mars/learn/contrib/pytorch/__init__.py

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 from .run_script import run_pytorch_script
+from .dataset import MarsDataset
```

Comment:
Add `# noqa: F401  # pylint: disable=unused-import` to end of this line to pass codacy.